yb-voyager/cmd/analyzeSchema.go

```diff
@@ -324,7 +324,7 @@ func addSummaryDetailsForIndexes() {
 
 func checkStmtsUsingParser(sqlInfoArr []sqlInfo, fpath string, objType string) {
 	for _, sqlStmtInfo := range sqlInfoArr {
-		_, err := queryparser.Parse(sqlStmtInfo.stmt)
+		_, err := queryparser.Parse(sqlStmtInfo.formattedStmt)
```

Comment:
was there any issue with stmt (unformmated) one being used before.

Moreover I think formattedStmt is safer choice, since originally it is present in that form, we do the unformmating by concat the string without newlines

Reply:
> was there any issue with stmt (unformmated) one being used before.

Not that I know of, but there was a discrepancy in this Parse() call and another call of it in the `ParseRequiredDDLs`, where we are using the `formattedStmt` so just to maintain consistency in both I have changed so that even if formattedStmt has some issue this `Parse` should catch it.